blog/content/post/2020-08-19-system_latency.md

```diff
@@ -0,0 +1,222 @@
+---
+author: jakub.lecki
```

Comment:
@leckijakub missing some crucial metadata in the header, check the template in the `blog/content/post/` directory

Reply:
Missing metadata added